lnbits/core/templates/core/core_admin.html

```diff
@@ -0,0 +1,717 @@
+{% extends "public.html" %} {% from "macros.jinja" import window_vars with
```

Comment:
- how can I view & test this `html` file?

Reply:
@talvasconcelos

Reply:
dead code